cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -26,19 +26,584 @@ Cask Data Application Platform Release Notes
 `Release 3.2.0 <http://docs.cask.co/cdap/3.2.0/index.html>`__
 =============================================================
 
+New Features
+------------
+
+- `CDAP-1914 <https://issues.cask.co/browse/CDAP-1914>`__ -
+  Added documentation of the RESTful endpoint to retrieve the properties of a stream.
+  
+- `CDAP-2556 <https://issues.cask.co/browse/CDAP-2556>`__ -
+  Added support for HBase1.1.
+  
+- `CDAP-2666 <https://issues.cask.co/browse/CDAP-2666>`__ -
+  Added a new API for creating an application from an artifact.
+  
+- `CDAP-2756 <https://issues.cask.co/browse/CDAP-2756>`__ -
+  Added the ability to write to multiple outputs from a MapReduce job.
+  
+- `CDAP-2757 <https://issues.cask.co/browse/CDAP-2757>`__ -
+  Added the ability to dynamically write to multiple partitions of a PartitionedFileSet
+  dataset as the output of a MapReduce job.
+  
+- `CDAP-2951 <https://issues.cask.co/browse/CDAP-2951>`__ -
+  Removed deprecated methods in cdap-client.
+  
+- `CDAP-3253 <https://issues.cask.co/browse/CDAP-3253>`__ -
+  Added a Stream and Dataset Widget to the CDAP-UI.
+  
+- `CDAP-3390 <https://issues.cask.co/browse/CDAP-3390>`__ -
+  Added stream views, enabling reading from a single stream using various formats and
+  schemas.
+  
+- `CDAP-3419 <https://issues.cask.co/browse/CDAP-3419>`__ -
+  HBaseQueueDebugger now shows the minimum queue event transaction write pointer both for
+  each queue and for all queues.
+  
+- `CDAP-3476 <https://issues.cask.co/browse/CDAP-3476>`__ -
+  Added a Validator Transform that can be used to validate records based on a set of
+  available validators and configured to write invalid records to an error
+  dataset.
+  
+- `CDAP-3514 <https://issues.cask.co/browse/CDAP-3514>`__ -
+  Added a Query HTTP RESTful API endpoint that retrieves a count of active queries:
```

Comment:
actually its weird. the PR is merged. Maybe we should just close the bug, and this should stay. @jwang47 ?